src/Workspaces/Core/Portable/Utilities/CancellationTokenExtensions.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Shared.Utilities
+{
+    /// <summary>
+    /// Extensions to <see cref="CancellationToken"/>.
+    /// </summary>
+    internal static class CancellationTokenExtensions
```

Comment:
Tests? If we're copying code we should be copying tests too.

Reply:
➡️ This type has now been removed